pkg/network/node/metrics/metrics.go

```diff
@@ -1,6 +1,6 @@
 // +build linux
 
-package node
+package metrics
```

Comment:
To the extent possible, you should be backporting only the new change you made. So don't backport moving `metrics.go` into a different directory

Reply:
All back-port included this exact patch:

https://github.com/openshift/sdn/pull/155/files
https://github.com/openshift/sdn/pull/154/files

So I am not sure what you're referring to?